manifests/nodejs.yml

```diff
@@ -287,13 +287,14 @@ tests/:
       Test_V2_Login_Events_Anon: missing_feature
       Test_V2_Login_Events_RC: missing_feature
     test_blocking_addresses.py:
-      Test_BlockingAddresses: *ref_3_19_0
       Test_BlockingGraphqlResolvers:
         '*': *ref_4_22_0
         nextjs: irrelevant # nextjs is not related with graphql
+      Test_Blocking_client_ip: *ref_3_19_0
```

Comment:
@uurien client ip blocking works with nextjs ?

Reply:
it should, it uses `http` module internally. it was working in removed line: `Test_BlockingAddresses: *ref_3_19_0`